earth_enterprise/src/server/wsgi/serve/push/search/util/search_schema_parser.py

```diff
@@ -123,9 +123,9 @@ def Parse(self, search_file, table_name, file_prefix=None):
       logger.info("File prefix is '%s'", file_prefix)
     self.__StartDocument()
     try:
-      context = xml.etree.cElementTree.iterparse(search_file,
+      context = ET.iterparse(search_file,
```

Comment:
Minor: maybe ought to clean up formatting here. With the shorter function call you can just put `SearchSchemaParser.EVENTS` on the same line.